Documentation/docs/source/IO.rst

```diff
@@ -0,0 +1,91 @@
+Image and Transformation IO
+***************************
+
+.. _image-io:
+
+Images
+======
+
+The default SimpleITK ``ImageFileReader`` can handle numerous file types using
```

Comment:
We should introduce what an ImageIO is little, consider working in something like the following:

There are numerous file formats support by SimpleITK's image readers and writers. Support for a particular format is handled by a specific ITK `ImageIO`(link) class.  By default, the ImageIO is automatically determined for a particular file. Advanced SimpleITK installations can configure or extend which file formats are supported by SimpleITK.